src/js_native_api.h

```diff
@@ -548,6 +548,7 @@ node_api_post_finalizer(node_api_basic_env env,
                         void* finalize_data,
                         void* finalize_hint);
 
+#define NODE_API_EXPERIMENTAL_HAS_FLOAT16_ARRAY
```

Comment:
It is rather unexpected to see this macro defined here. It must be defined in the same place where you define the new enum value. E.g:
```cpp
#ifdef NAPI_EXPERIMENTAL
#define NODE_API_EXPERIMENTAL_HAS_FLOAT16_ARRAY
  napi_float16_array,
#endif // NAPI_EXPERIMENTAL
```